winml/adapter/ZeroCopyInputStreamWrapper.cpp

```diff
@@ -11,7 +11,8 @@ using namespace Windows::AI::MachineLearning;
 
 // ZeroCopyInputStreamWrapper
 ZeroCopyInputStreamWrapper::ZeroCopyInputStreamWrapper(
-    wss::IRandomAccessStreamReference stream) : stream_(stream) {
+    ABI::Windows::Storage::Streams::IRandomAccessStreamReference* stream) {
+    winrt::copy_from_abi(stream_, (void*)stream);
```

Comment:
just confirming - even though "copy" is in the name, this doesn't do a copy, right? from the [doc](https://docs.microsoft.com/en-us/uwp/cpp-ref-for-winrt/copy-from-abi) , " handle, or a raw pointer to a target whose lifetime should be managed by the C++/WinRT object. In the case of the IUnknown&, void* overload, the function calls AddRef on value." so it will just addref right?

Reply:
right !  this is a "Copy" of the point which means it gets a ref count.   it doesn't copy the stream content, it copies the point.  copy gives a strong ref vs. a weak ref.  fun right?

---
In reply to: [354041543](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354041543) [](ancestors = 354041543)